src/config/custom.js

```diff
@@ -1,5 +1,5 @@
 'use strict'
 
 module.exports = {
-  timeout: 80 * 1000
+  timeout: 40 * 1000
```

Comment:
reduced to `5 seconds` and we get a red flag on node tests for slow tests:

<img width="357" alt="screen shot 2017-07-09 at 17 38 16" src="https://user-images.githubusercontent.com/790842/27995376-6f11ad20-64cd-11e7-998c-21757b34bab9.png">